parser.c

```diff
@@ -889,7 +901,8 @@ static Ast *read_if_stmt(void)
     expect(')');
     Ast *then = read_stmt();
     Token *tok = read_token();
-    if (!tok || get_ttype(tok) != TTYPE_IDENT || strcmp(get_ident(tok), "else")) {
```

Comment:
Don't touch this line since there is no functional change.

Reply:
But this change is made by command "clang-format -i *.[ch]". Should I maintain the style in ordinary file? (There also exist changes in lexer.c and mzcc.h).

Reply:
I suffered from style inconsistency problem by different versions of `clang-format`. The version I am using is `8.0.0`, and I would suggest to minimize the changes.